docs/user_guide/transformation/PowerTransformer.rst

```diff
@@ -5,66 +5,349 @@
 PowerTransformer
 ================
 
-The :class:`PowerTransformer()` applies power or exponential transformations to numerical
-variables.
+Power transformations are a family of mathematical functions used to transform numerical variables into a more suitable shape for modeling. The transformation function is typically represented as :math:`x' = x^\\lambda`, where :math:`x` is the original variable and :math:`\\lambda` is the transformation parameter.
```

Comment:
the exponential math function is not rendering as it should. Did you try building the docs?

Reply:
I built it to see the tests, but I haven't checked it rendered yet. I'm checking it now and arranging the fixes. Thx for the heads up!